confgenerator/testdata/valid/linux/logging-custom_use_build_in_receivers/input.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2020 Google LLC
```

Comment:
Nit: `…built_in_receivers`.

Reply:
Also, this is more than logging… `all-custom_use_built_in_receivers`?

Reply:
Done! Thanks